src/App/Document.cpp

```diff
@@ -3817,7 +3822,12 @@ DocumentObject * Document::getActiveObject() const
     return d->activeObject;
 }
 
-DocumentObject * Document::getObject(const char *Name) const
+DocumentObject * Document::getObject(const char* Name) const
+{
+    return getObject(std::string(Name));
+}
+
+DocumentObject * Document::getObject(const std::string Name) const

```

Comment:
Argument should be `const std::string&` as otherwise it creates a tmp. std::string